src/regressionmodel.jl

```diff
@@ -25,13 +25,18 @@ If this column does not exist, it is created.
 Pointwise standard errors are written into the column specified by `err_col`.
 
 !!! note
-    Effects are computed on the scale of the transformed response.
-    For models with an explicit transformation, that transformation
-    is the scale of the effects. For models with a link function,
-    the scale of the effects is the _link_ scale, i.e. after
-    application of the link function. For example, effects for
-    logitistic regression models are on the logit and not the probability
-    scale.
+    By default (`invlink=identity`), effects are computed on the scale of the
+    transformed response. For models with an explicit transformation, that
+    transformation is the scale of the effects. For models with a link function,
+    the scale of the effects is the _link_ scale, i.e. after application of the
+    link function. For example, effects for logitistic regression models are on
+    the logit and not the probability scale.
```

Comment:
why is this the default when the link can (in principle) be obtained from the model itself?  to avoid a dependency on GLM?

Reply:
Exactly, to avoid the dependency on GLM. As part of the associated docs issue (#38), I think it would be good to have a GLM example. If we decide to take GLM as a dependency, then we should add more specialized methods that take advantage of `GLM.mueta` rather than using AD for the derivative.